js_modules/dagster-ui/packages/ui-core/src/runs/RunTimingTags.tsx

```diff
@@ -0,0 +1,59 @@
+import {Box, Popover, Tag} from '@dagster-io/ui-components';
+import * as React from 'react';
+
+import {formatElapsedTime} from '../app/Util';
+import {TimestampDisplay} from '../schedules/TimestampDisplay';
+
+import {RunTimingDetails} from './RunTimingDetails';
+import {RunTimingFragment} from './types/RunTimingDetails.types';
+
+export const RunTimingTags = ({loading, run}: {loading: boolean; run: RunTimingFragment}) => {
```

Comment:
Just moved these timing tags to their own file so there's a bit less going on in RunRoot